common/source/docs/common-ice.rst

```diff
@@ -103,7 +101,22 @@ QuadPlanes can also automatically stop the engine once the final phase of a VTOL
 
 .. note:: The engine can be started and stopped in autonomous missions by using the ``DO_ENGINE_CONTROL`` mission command. This useful preceding NAV_VTOL_TAKEOFF or NAV_VTOL_LAND to start or stop the engine to prevent prop strikes while landing in wind with a low clearance prop. See :ref:`common-mavlink-mission-command-messages-mav_cmd` section. 
 
+Idle and redline governors
+--------------------------
+
+The idle governor allows the autopilot to adjust the throttle to maintain an RPM value when the commanded throttle is low. This can be useful when the engine is waiting for takeoff and reduces the workload on the pilot during that time. Increasing the throttle command will give the expected throttle response.
+
+- :ref:`ICE_IDLE_PCT<ICE_IDLE_PCT>`
```

Comment:
explanations of each param?

Reply:
In the param list.

Reply:
still should have it here also...some arent even aware there is a param list...